rero_ils/theme/templates/rero_ils/page_wiki.html

```diff
@@ -37,7 +39,10 @@
 
 {%- block javascript %}
 {{ super() }}
-<script type="text/javascript" src="{{ url_for('wiki.static', filename='js/wiki.js') }}"></script>
+<!-- EasyMDE -->
+<script src="https://cdn.jsdelivr.net/npm/easymde/dist/easymde.min.js"></script>
+<script type="text/javascript" src="{{ url_for('wiki.static', filename='js/wiki.js') }}">
```

Comment:
I suggest to fix the version of easymde to avoid unexpected results after an update.

Reply:
Done